packages/lib/src/prod/remote-production.ts

```diff
@@ -165,6 +165,25 @@ export {__federation_method_ensure, __federation_method_getRemote , __federation
       if (builderInfo.isRemote) {
         for (const expose of parsedOptions.prodExpose) {
           if (!expose[1].emitFile) {
+            const resolveId = await this.resolve(expose[1].id)
```

Comment:
The transform method will run very many times, especially on larger libraries, can it be changed like this?
``` js
if(!expose[1].id) expose[1].id = (await this.resolve(expose[1].import))?.id;
```
Of course, before doing so, we need to remove the assignment to `expose[1].id`
https://github.com/originjs/vite-plugin-federation/blob/main/packages/lib/src/prod/expose-production.ts#L30

Reply:
@ruleeeer yes, it look like better. I will solved it soon :-)